### DISCUSSION TITLE
turn off "enable_autosnippets"

### DISCUSSION THREAD

**Discussion by JohnHelvetius** [L3MON4D3/LuaSnip#1134] [Q&A] [answered]
How to temporarily set the config option of "enable_autosnippets" to false at the command line of neovim?

**Comment by L3MON4D3** [comment]
Oh, we don't have that :/
But it might be a good idea to include, I'll think about a mechanism to update the config or something like that :D

**Reply by L3MON4D3** [reply, marked as answer]
(you can manually achieve this by storing your config, swapping that one key, and then setting it again via `ls.setup`)
